Upstream-Status: Inappropriate

GN requires libstdc++.a to be installed on host systems by default,
which is often not very common, for no particular reason.

Signed-off-by: Raphael Kubo da Costa <raphael.kubo.da.costa@intel.com>
Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 58bcefc659a25cd88c1c26a9eaa99bf217124df3 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Thu, 19 Dec 2019 15:47:53 +0200
Subject: [PATCH] Pass no-static-libstdc++ to gen.py

---
 tools/gn/bootstrap/bootstrap.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 8e121b59f387..c61c92bd22c7 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -110,6 +110,7 @@ def main(argv):
   cmd = [
       sys.executable,
       os.path.join(GN_ROOT, 'build', 'gen.py'),
+      '--no-static-libstdc++',
       '--no-last-commit-position',
       '--out-path=' + gn_build_dir,
   ]
-- 
2.20.1

